### PR TITLE
Run black on all files

### PR DESCRIPTION
**Pull request by gaffney2010**
Fixes https://github.com/Axelrod-Python/Axelrod/issues/1312

First commit runs black on all files.  Second commit adds a check to config.yml.

I didn't make any changes to the contribution docs, because they already say that you should run black on all files.

### CODE REVIEW

**Comment by marcharper** [comment]
I didn't think there would be so many changes to so many files, do you mind holding off until after I finish #1288 ? I'm assuming that you would just have to run black again and that wouldn't create much extra effort.

**Comment by gaffney2010** [comment]
@marcharper That's fine with me.

**Comment by drvinceknight** [comment]
> I didn't think there would be so many changes to so many files, do you mind holding off until after I finish #1288 ?

I was surprised when you suggested it would be fine :) :+1:

File: docs/tutorials/contributing/guidelines.rst [hunk 1]
```diff
@@ -12,8 +12,11 @@ The project follows the following guidelines:
    <https://www.python.org/dev/peps/pep-0008/>`_ which includes **using
    descriptive variable names**.
 3. Code Format: Use the `Black formatter <https://github.com/ambv/black>`_ to format
-    all code  and the `isort utility <https://github.com/timothycrosley/isort>`_ to
-    sort import statements.
+   all code  and the `isort utility <https://github.com/timothycrosley/isort>`_ to
+   sort import statements.  You can run black on all code with::
+
+    $ python -m black -l 80 .
```
Comment (drvinceknight) [inline docs/tutorials/contributing/guidelines.rst:18]: Thanks for this TJ :+1:

**Comment by marcharper** [comment]
Closing in favor of #1368

**Pull request closed without merging**